source/include/ceph_troubleshooting.rst

```diff
@@ -26,41 +32,6 @@ The failed hardware device is logged by the Linux kernel:
 Cross-reference the hardware device and OSD ID to ensure they match.
 (Using `pvs` and `lvs` may help make this connection).
 
-Removing a Failed Ceph Drive
```

Comment:
Does it need to move to ceph-ansible?

Reply:
It's also in Ceph-ansible (added by #34).

Reply:
This section isn't in the ceph-ansible docs in #34. If it's specific to ceph-ansible then it probably needs to move to source/include/ceph_ansible.rst

Reply:
Right, readded,